src/msgraph_core/requests/batch_request_builder.py

```diff
@@ -31,7 +31,7 @@ def __init__(
         if request_adapter is None:
             raise ValueError("request_adapter cannot be Null.")
         self._request_adapter = request_adapter
-        self.url_template = f"{self._request_adapter.base_url}/$batch"
+        self.url_template = f"{self._request_adapter.base_url}$batch"
```

Comment:
```suggestion
        self.url_template = f"{self._request_adapter.base_url.removesuffix('/')}/$batch"
```
Would it be better to remove the suffix incase a user sets a custom base url without the `/`